aquarius/events/processors.py

```diff
@@ -196,9 +209,25 @@ def process(self):
             raise Exception("RBAC permission denied.")
 
         _record = self.make_record(asset)
+
         if _record:
             try:
                 record_str = json.dumps(_record)
+                try:
```

Comment:
Three things:
1. I don't like the try inside a try. It looks funky and it could mess up exception handling. There has to be another way.
2. We shouldn't set up exceptions as part of the flow. Use conditionals. There is also the optioin of working inside the _es_instance update function, e.g. adding a write_over parameter or something similar. But this code needs to be imprived, most definitely.
3. See that in v4main I removed the created assignment and logging since that is obsolete code left there by mistake, and will mess up the processing :)

Reply:
this is still an issue